saleor_gql_loader/utils.py

```diff
@@ -55,6 +58,45 @@ def graphql_request(query, variables={}, headers={},
         return parsed_response
 
 
+def graphql_multipart_request(body, headers, endpoint=GQL_DEFAULT_ENDPOINT):
+    """Execute a multipart graphQL query with `body` provided on the `endpoint`.
+
+    Parameters
+    ----------
+    body : str
+        payloads of graphQL query.
+    headers : dict, optional
+        headers added to the request (important for authentication).
+    endpoint : str, optional
+        the graphQL endpoint url that will be queried, default is
+        `GQL_DEFAULT_ENDPOINT`.
+
+    Returns
+    -------
+    response : dict
+        a dictionary corresponding to the parsed JSON graphQL response.
+
+    Raises
+    ------
+    Exception
+        when `response.status_code` is not 200.
+    """
+    bodyEncoder = MultipartEncoder(body)
+    base_headers = {
+        "Content-Type": bodyEncoder.content_type,
+    }
+    override_dict(headers, base_headers)
```

Comment:
I think here we want to override the `base_headers` with the `headers` not the other way arround ?